ubuntudesign/vanilla-brochure-theme
# Update styling on  media object pattern

## Done

- Remove margin-top from image
- On small screens image should be 60px h/w - otherwise, 72px

## QA

- Sync branch
- Run `gulp jekyll`
- View [http://127.0.0.1:4001/vanilla-brochure-theme/examples/patterns/media-object/](http://127.0.0.1:4001/vanilla-brochure-theme/examples/patterns/media-object/)
- Should match[ design spec](https://raw.githubusercontent.com/ubuntudesign/vanilla-brochure-theme-design/master/Media%20object/media-object.png)

Fixes #21
